Tests/CssTests/FullStylesheetTests.swift

```diff
@@ -9,15 +9,15 @@ class FullStylesheetTests: XCTestCase {
   func testABigStyleSheet() {
     let css: Stylesheet =
       body % (
-        .background(red)
+        .background(Color.red)
```

Comment:
Background color needs to be qualified now, though this speaks maybe more to the fact that the attribute accepting only `color` is `background-color`.